### PR TITLE
Fix pending state for limit orders

### LINKED ISSUE #1488: Remove 'Pending' state when limit order is opened
Discussion: https://cowservices.slack.com/archives/C0361CDG8GP/p1669218115067039

No need to show 'Pending' status when a Limit order is opened.

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes #1488 

This PR will remove pending indicator from the header status component when there are open limit orders and will only show pending indicator for the open swap orders.

## To test
- create a limit order
- there should be no pending indicator in the header status field
- create a swap order
- there should be pending indicator as usual

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
I haven't reviewed the code yet. Commenting on the test I Did.

I did one swap, and one limit order, I get to this state:

image

Why is the spinner not in the header if I still have a swap pending?

File: src/custom/state/orders/actions.ts [hunk 1]
```diff
@@ -21,6 +21,11 @@ export enum OrderStatus {
   REFUNDED = 'refunded',
 }
 
+export enum OrderClass {
```
Comment (shoom3301) [inline src/custom/state/orders/actions.ts:24]: https://github.com/cowprotocol/cow-sdk/commit/0a99544ae5d9d4ff39e807a3ffdf416f14b455e1#diff-7bb820318d1e4352d1cd71225f5a30101b8b30cf1186af9b405eecc654e3ee68R13

I have updated the SDK, so we can import the type from it

Comment (shoom3301) [reply src/custom/state/orders/actions.ts:24]: [v1.0.1-RC.2](https://github.com/cowprotocol/cow-sdk/releases/tag/v1.0.1-RC.2)

**Comment by nenadV91** [comment]
@anxolin That is not related to this PR, its just a style issue on dark mode where the loader color was black. But I've added a fix same as it is on develop right now.

File: src/custom/state/orders/actions.ts (hunk 1 above)
Comment (nenadV91) [reply src/custom/state/orders/actions.ts:24]: If I update the cow-sdk, then we get into the issues of supported chains, and I think we should do that in a separate PR and leave this one as is.

**Comment by elena-zh** [comment]
Hey @nenadV91 , great job!
However, I noticed that Open Limit orders do not appear in the Account modal. 
They appear only when they are cancelled/Expired.

**Review by shoom3301 — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]
LGTM now!

File: src/cow-react/api/gnosisProtocol/api.ts [hunk 2]
```diff
@@ -32,6 +32,7 @@ import { getAppDataHash } from 'constants/appDataHash'
 import { Context } from '@sentry/types'
 import { PriceInformation, SimpleGetQuoteResponse } from '@cowprotocol/cow-sdk'
 import { GpPriceStrategy } from 'state/gas/atoms'
+import { OrderClass } from '@src/custom/state/orders/actions'
```
Comment (anxolin) [inline src/cow-react/api/gnosisProtocol/api.ts:35]: delete `@src/custom`

**Review by anxolin — APPROVED** [review]
@nenadV91 pls, add some more details in the description. i don't think is clear what it does just by reading it

**Pull request merged**